Services/Ordering/Ordering.Domain/ValueObjects/Address.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Ordering.Domain.ValueObjects
+{
+    public class Address
+    {
+        private string _street;
+        private string _city;
+        private string _state;
+        private string _country;
+        private string _zipCode;
+
+        public Address(string street, string city, string state, string country, string zipCode)
+        {
+            _street = !string.IsNullOrWhiteSpace(street) ? street : throw new DomainException($"Invalid {nameof(street)}");
+            _city = !string.IsNullOrWhiteSpace(city) ? city : throw new DomainException($"Invalid {nameof(city)}");
+            _state = !string.IsNullOrWhiteSpace(state) ? state : throw new DomainException($"Invalid {nameof(state)}");
+            _country = !string.IsNullOrWhiteSpace(country) ? country : throw new DomainException($"Invalid {nameof(country)}");
+            _zipCode = !string.IsNullOrWhiteSpace(zipCode) ? zipCode : throw new DomainException($"Invalid {nameof(zipCode)}");
+        }
+
+        public string GetStreet()
```

Comment:
can be simplified using property with getter only (it still can be assigned from ctor)